src/mon/PGMonitor.cc

```diff
@@ -236,8 +236,12 @@ void PGMonitor::upgrade_format()
 void PGMonitor::post_paxos_update()
 {
   dout(10) << __func__ << dendl;
-  if (mon->osdmon()->osdmap.get_epoch()) {
-    send_pg_creates();
+  OSDMap& osdmap = mon->osdmon()->osdmap;
+  if (osdmap.get_epoch()) {
+    if (osdmap.get_num_up_osds() > 0) {
+      assert(osdmap.get_up_osd_features() & CEPH_FEATURE_MON_STATEFUL_SUB);
+    }
+    check_subs();
```

Comment:
could move `check_subs()` into the `if (osdmap.get_num_up_osds() > 0)` block, because only OSD is interested in the pg creates updates.